test/test_knn.py

```diff
@@ -57,3 +57,35 @@ def test_knn_graph(dtype, device):
     row = row.view(-1, 2).sort(dim=-1)[0].view(-1)
     assert row.tolist() == [1, 3, 0, 2, 1, 3, 0, 2]
     assert col.tolist() == [0, 0, 1, 1, 2, 2, 3, 3]
+
+
+@pytest.mark.parametrize('dtype,device', product(grad_dtypes, devices))
+def test_knn_graph_large(dtype, device):
+    d = pickle.load(open("test/knn_test_large.pkl", "rb"))
```

Comment:
Do we really need these files to be added to the library? How about just using `torch.randn(...)`?

Reply:
That could work, and do we just compute the expected result by the previous `numpy` implementation?

Reply:
Sounds good :)

Reply:
I will address all of the issues probably sometime this week (more likely Saturday-Sunday), But if you feel like doing them, you are always welcomed!

Reply:
Sounds good, I will try to help you with those. Let's aim for a merge in the next week :)